src/main/java/cs/ualberta/cmput402/tictactoe/board/ScoreBoard.java

```diff
@@ -0,0 +1,33 @@
+package cs.ualberta.cmput402.tictactoe.board;
+
+public class ScoreBoard {
+	private int xWin;
+	private int oWin;
+	private int tie;
+	
+	public ScoreBoard() {
+		xWin = 0;
+		oWin = 0;
+		tie = 0;
+	}
+	
+	public void setXwin() {
+		xWin++;
+	}
+	
+	public void setOwin() {
+		oWin++;
+	}
+	
+	public void setTie() {
+		tie++;
+	}
+	
+	public void printScoreBoard() {
+		System.out.println("*****************************************************");
+		System.out.printf("Player X won: %d games, lost %d games\n",xWin, oWin);
+		System.out.printf("Player O won: %d games, lost %d games\n",oWin, xWin);
+		System.out.printf("Number of ties: %d\n", tie);
+		System.out.println("*****************************************************");
+	}
+}
```

Comment:
The approach to achieve feature 2 does not have any problems and the code is good. We only have two suggestions.

1. It seems that you choose to follow the lower camel case naming convention in the project. Strictly speaking, the function name should be "setXWin" and "setOWin" rather than "setXwin" and "setOwin".

2. It would be better to have some documentation and comments for methods when you create a new class.

[This is just a suggestion to improve the code and the pull request message]

Reply:
1. This is addressed by commit ae6eb3ef99269e6429767a62535eb7795af882dd

2. This has been addressed by commit c3022772954bcc28cbcfd9bad6d66f43452017db